Frontend/ui-customer/src/Flow.purs

```diff
@@ -512,6 +514,15 @@ updateCustomerVersion dbClientVersion dbBundleVersion = do
     else pure unit
   else pure unit
 
+makeAppInstallsCall :: String -> String -> FlowBT String Unit
+makeAppInstallsCall versionName bundle = do
+  let sourceUrl = getValueToLocalNativeStore REFERRER_URL
+      Version clientVersion = stringToVersion versionName
+      Version bundleVersion = stringToVersion bundle
+  (AppInstallsRes resp) <- Remote.appInstallsBT (Remote.makeAppInstallsReq (sourceUrl) (Version clientVersion) (Version bundleVersion))
+  setValueToLocalNativeStore REFERRER_DATA (if resp.result == "Success" then "API_SUCCESS" else "API_FAILED")
+  pure unit
```

Comment:
After makeAppInstallsCall function is called, where is the flow pointing to? if it's not a separate flow then add this function within the function where it's called under `where` block